script/publish

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Releasing gem
+bundle exec rake release
+
+# Releasing NPM package
+npm publish --access public
```

Comment:
Will this require the dev to have a token or something locally?

Reply:
This requires the dev to be authorized to NPM locally (and the user needs to have access to `@primer` org)
When running `npm publish` the script tell you what to do if you are missing something